lists/opengl.md

```diff
@@ -30,16 +30,22 @@
 
 - #### [OpenGL 4 Reference](https://www.khronos.org/registry/OpenGL-Refpages/gl4//) [N/A]
 	- Quick reference index for modern OpenGL including version 3.3. Also contains references for GLSL.
+	
+- #### [OpenGL Programming Guide: The Official Guide to Learning OpenGL, Version 4.5 with SPIR-V (9th Edition) ](https://www.amazon.com/OpenGL-Programming-Guide-Official-Learning/dp/0134495497)
+	- Great book for learning OpenGL up to 4.5.
 ### Techniques
 
 
 
 
 
 ### Tutorials
-
-
-
+- #### [OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP)
```

Comment:
Don't forget to include difficulty level alongside link. "[OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP) [Beginner]", Difficulty can be either Beginner/Intermediate/Advanced or N/A.